tests/test_player.py

```diff
@@ -2,27 +2,24 @@
 import unittest
 from game.player import Player
 from game.gameboard import GameBoard
+from game.strategy_one import StrategyOne
+from mock import MagicMock
 
 class TestPlayer(unittest.TestCase):
     def setUp(self):
-        self.player_blue = Player('blue')
-        self.player_red = Player('red')
+        strategy_dummy = StrategyOne()
+        strategy_dummy.return_column = MagicMock(return_value = 1)
+        self.player_blue = Player('blue', strategy_dummy)
 
-    def test_player_with_default_strategy(self):
-        board = GameBoard.from_matrix([
-                    ['red',  'blue',  'red',  'red', 'blue',   None,  None],
-                    ['red',   'red', 'blue',  'red',  'red',   None,  None],
-                    ['red',  'blue',  'red',  'red', 'blue', 'blue',  None],
-                    ['blue', 'blue',  'red', 'blue',  'red',  'red', 'blue'],
-                    ['red',  'blue',  'red', 'blue',  'red', 'blue',  'red'],
-                    ['blue',  'red', 'blue',  'red',  'red',  'red',  'red']])
+    def test_player_play_with_dummy_strategy(self):
+        board = GameBoard() # empty board
         expected_board = GameBoard.from_matrix([
-                    ['red',  'blue',  'red',  'red', 'blue',   None,  None],
-                    ['red',   'red', 'blue',  'red',  'red', 'blue',  None],
-                    ['red',  'blue',  'red',  'red', 'blue', 'blue',  None],
-                    ['blue', 'blue',  'red', 'blue',  'red',  'red', 'blue'],
-                    ['red',  'blue',  'red', 'blue',  'red', 'blue',  'red'],
-                    ['blue',  'red', 'blue',  'red',  'red',  'red',  'red']])
+                    [None,   None, None, None, None, None, None],
+                    [None,   None, None, None, None, None, None],
+                    [None,   None, None, None, None, None, None],
+                    [None,   None, None, None, None, None, None],
+                    [None,   None, None, None, None, None, None],
+                    ['blue', None, None, None, None, None, None]])
         self.player_blue.play(board)
         self.assertEqual(board, expected_board)
```

Comment:
They are slightly different cases. What you want to test here is how the player reacts based on what  column the strategy returns. By being able to mock the strategy you can manage this as you please, hence being able to easily generate all the cases mentioned above. 

On the other hand, you can (and should) test the strategy without even creating a player; just set up the board and send the `return_column` message. But there you will be testing that the strategy actually does what you expect it to do (so, if it works ok, it should never return things like 0 or 8 :))